test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js

```diff
@@ -46,4 +48,25 @@ export default class ChallengeOktaVerifyPushPageObject extends ChallengeFactorPa
     return this.form.getElement(FORM_INFOBOX_WARNING);
   }
 
+  async autoChallengeInputExists(selector) {
```

Comment:
nit: I was thinking about this a little more, and what we could do is add a selector like `$('[name$="autoChallenge"]` (attribute ends with) here, so we wouldn't need to pass the selector. Maybe give it a try?
```
return this.form.elementExist('[name$="autoChallenge"]');
```